### ISSUE TITLE
`simplify()` returns an evaluated value with certain expressions containing `n`

### ISSUE THREAD

**Issue by tanman-** [josdejong/mathjs#949]
The expression patterns are similar to those in #948 .

### Not OK
The `n` mixed with `-` seems to be an issue. `n` is somehow set to `8`.
```js
math.simplify('8 - 4n').toString(); // -24
math.simplify('8 - n').toString(); // 0
```

### OK
```js
math.simplify('8 - 4q').toString(); // 8 - 4 * q
math.simplify('8 + 4n').toString(); // 4 * n + 8
math.simplify('8 / 4n').toString(); // 2 * n
math.simplify('8 * 4n').toString(); // 32 * n
math.simplify('8 - 4nq').toString(); // 8 - 4 * nq
```

### Workaround
```js
simplify('8 - 4n', []).toString()
```
`[]`: no rules applied.

**Comment by josdejong** [comment]
Thanks for reporting this bug!

**Comment by josdejong** [comment]
When I try `math.simplify('8 - 4n1')`, using an other internally used symbol `n1` instead of `n`, we get an infinite loop.

This issue probably has the same underlying issue as #948, let's see.

**Comment by ericman314** [comment]
Resolved by #950

**Comment by josdejong** [comment]
This should be fixed now in `v3.16.4`
